resqpy/time_series.py

```diff
@@ -364,6 +376,31 @@ def time_series_from_list(timestamp_list, parent_model = None):
       time_series.add_timestamp(timestamp)
    return time_series
 
+def merge_timeseries_from_uuid(model, timeseries_uuid_iter):
+   """Create a TimeSeries object from an iteratable object of existing timeseries UUIDs of timeseries. iterable can be a list, array, or iteratable generator (model must be provided). The new timeseries is sorted in ascending order. Returns the new time series, the new time series uuid, and the list of timeseries objects used to generate the list"""
+   
+   reverse=False 
+
+   alltimestamps=set({})
+   timeserieslist=[]
+   for timeseries_uuid in timeseries_uuid_iter:
+       timeseriesroot=model.root(uuid=timeseries_uuid)
+       assert(rqet.node_type(timeseriesroot) == 'obj_TimeSeries')
+       
+       singlets=TimeSeries(model, time_series_root=timeseriesroot)
+       timeserieslist.append(singlets)
+       #alltimestamps.update( set(singlets.timestamps) )
+       alltimestamps.update( set(singlets.datetimes()) )
+   
+   if reverse is None:
```

Comment:
Will reverse ever be None? It seems to be defined as False above

Reply:
No it won't. I originally tried to allow creation of a timeseries with an argument sorting or without sorting, but it seems like the returned timeseries generated from a list of a timestamps is almost always in ascending sorted order. So I removed the (reverse) argument.